docs/debugging.md

```diff
@@ -57,6 +57,8 @@ To debug the JavaScript code in Chrome, select "Debug JS Remotely" from the Deve
 
 Select `Tools → Developer Tools` from the Chrome Menu to open the [Developer Tools](https://developer.chrome.com/devtools). You may also access the DevTools using keyboard shortcuts (`⌘⌥I` on macOS, `Ctrl` `Shift` `I` on Windows). You may also want to enable [Pause On Caught Exceptions](http://stackoverflow.com/questions/2233339/javascript-is-there-a-way-to-get-chrome-to-break-on-all-errors/17324511#17324511) for a better debugging experience.
 
+> Note: on Android, if the times between the debugger and device have drifted; things such as animation, event behavior, etc., might not work properly or the results may not be accurate. Please correct this by running `` adb shell "date `date +%m%d%H%M%Y.%S`" `` on your debugger machine. Root access is required for the use in real device.
```

Comment:
@safaiyeh updated. kindly check

Reply:
cc @AndrewMorsillo

Reply:
Maybe change the command to `` adb shell "date `date +%m%d%H%M%Y.%S%3N`" `` (aadd %3N to the end) to also sync milliseconds. Without that time can still be off by up to 999ms which can cause issues and performance problems.

Reply:
Thanks for the change @gedeagas, +1 to @AndrewMorsillo suggestion

Reply:
@AndrewMorsillo @safaiyeh updated.

Reply:
I am also updating the adb command on react-native emitTimeDriftWarning 

Related PR. https://github.com/facebook/react-native/pull/29673/